### PR TITLE
WAITP-1461: Datatrak design feedback

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:
Refer to this ticket for the list of changes
https://linear.app/bes/issue/WAITP-1461/design-feedback-week-of-20th-october

--

### CODE REVIEW

File: packages/datatrak-web/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no maximum-scale=1"  />
```
Comment (tcaiger) [inline packages/datatrak-web/index.html:5]: Stop mobile browsers zooming on on form field focus

File: packages/datatrak-web/src/features/Questions/GeolocateQuestion/MapModal/Map.tsx [hunk 2]
```diff
@@ -14,10 +14,20 @@ import {
   ZoomControl,
 } from 'react-leaflet';
 
+// On small screens the map needs to fit in the body of the modal so that we don't create a scroll trap fo r the user
+// but the map container also needs a set height to render correctly so it's not possible to use flexbox to fill the height
+// and so we need to set the height manually using this fudge factor.
```
Comment (acdunham) [inline packages/datatrak-web/src/features/Questions/GeolocateQuestion/MapModal/Map.tsx:19]: Nice comments, thanks

File: packages/datatrak-web/src/types/mui.d.ts [hunk 3]
```diff
@@ -27,3 +27,12 @@ declare module '@material-ui/pickers' {
...
   };
 }
+
+declare module '@material-ui/core/styles/createPalette' {
+  interface Palette {
+    primaryHover: string;
+  }
+  interface PaletteOptions {
+    primaryHover: string;
+  }
```
Comment (acdunham) [inline packages/datatrak-web/src/types/mui.d.ts:37]: Nice solution to this

**Review by acdunham — APPROVED** [review]
LGTM, thanks for these changes!

**Pull request merged**